charts/rancher-logging-example/charts/log-output/templates/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "log-output.fullname" . }}
+  labels:
+    {{- include "log-output.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.port }}
+      targetPort: http
+      protocol: TCP
+      name: http
```

Comment:
If the port is change-able, will it always be http? (I think yes, but want to confirm)

Reply:
it will always be http.